cli/cli-flags.js

```diff
@@ -412,13 +412,13 @@ function coerceOutput(values) {
 
 /**
  * Verifies outputPath is something we can actually write to.
- * @param {unknown} value
+ * @param {unknown=} value
  * @return {string=}
  */
 function coerceOutputPath(value) {
   if (value === undefined) return;
 
-  if (typeof value !== 'string' || !fs.existsSync(path.dirname(value))) {
+  if (typeof value !== 'string' || !value || !fs.existsSync(path.dirname(value))) {
```

Comment:
In what situation will `value === undefined`.

If it's the case where `--output-path` is provided without any value, could we do a different error message for that?

Reply:
> In what situation will value === undefined.

when not provided. coerce is still called when param is not specified (or at least, it is for the "flags in a json path" case).

> If it's the case where --output-path is provided without any value, could we do a different error message for that?

I think the current error is sufficient.